guides/source/active_job_basics.md

```diff
@@ -123,6 +150,11 @@ module YourApp
 end
 ```
 
+NOTE: Since jobs run in parallel to your Rails application, most queuing implementations
+require that you start an implementation-specific queuing service (in addition to
+starting your Rails app) for the job processing to work. For information on
+how to do that refer to the documentation of your respective queuing implementation.
```

Comment:
s/queuing implementation/queuing library.

I also never remember if it's queuing or queueing.. such as above, we have "enqueuing".